CI/src/main/java/group1/email.java

```diff
@@ -0,0 +1,83 @@
+package group1;
+
+/*
+https://netcorecloud.com/tutorials/send-email-in-java-using-gmail-smtp/?hss_channel=tw-3300992070
+
+This code is not writen by group 1 it is copied line by line from the link abouve.
+Group 1 does not clame to have writen this code.
+*/
+
+
+import java.util.Properties;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+public class email {
+
+    public static void main(String[] args) {
+        // Recipient's email ID needs to be mentioned.
+        String to = "robin.wanlund@hotmail.com";
+
+        // Sender's email ID needs to be mentioned
+        String from = "group1SoffaKth@gmail.com";
+
+        // Assuming you are sending email from through gmails smtp
+        String host = "smtp.gmail.com";
+
+        // Get system properties
+        Properties properties = System.getProperties();
+
+        // Setup mail server
+        properties.put("mail.smtp.host", host);
+        properties.put("mail.smtp.port", "465");
+        properties.put("mail.smtp.ssl.enable", "true");
+        properties.put("mail.smtp.auth", "true");
+
+        // Get the Session object.// and pass username and password
+        Session session = Session.getInstance(properties, new javax.mail.Authenticator() {
+
+            protected PasswordAuthentication getPasswordAuthentication() {
+                //konto: group1SoffaKth@gmail.com
+                //losen: group1arbast!QWE
+                return new PasswordAuthentication("group1SoffaKth@gmail.com", "group1arbast!QWE");
```

Comment:
no passwords please

Reply:
change your password, make the code read the password from a file, and add this file to the gitignore